packages/scss/src/components/highlightedHeader/component.scss

```diff
@@ -0,0 +1,55 @@
+@use '@lucca-front/scss/src/commons/utils/namespace';
+@use '@lucca-front/scss/src/commons/utils/media';
+
+@mixin component($atRoot: namespace.$defaultAtRoot) {
+	overflow: hidden;
+	padding-inline: var(--pr-t-spacings-200);
+
+
+	@at-root ($atRoot) {
+		.highlightedHeader-content {
+			display: flex;
+			flex-direction: column;
+			margin: 0 auto;
+			min-height: 10rem;
+			min-width: min(20rem, 100%);
+			padding-block: var(--pr-t-spacings-300);
+			width: fit-content;
+			position: relative;
+		}
+
+		.highlightedHeader-content-box {
+			position: relative;
+			padding: var(--pr-t-spacings-300);
+			background-color: var(--pr-t-elevation-surface-raised);
+			border-radius: 1rem;
```

Comment:
Arf, on n'a pas de var globale pour les borders de 1rem ?
(je viens de vérifier mais du coup, normal qu'on a 1rem pour juste ce composant ?)

Reply:
This is planned for the future radius design token project. In the meantime, we've left it that way.

Reply:
Do we want the border radius to scale depending on the root font size? We usually define border radius values in pixels to avoid this.

Reply:
good point i change it thanks !